src/components/Table/utils/TableState.ts

```diff
@@ -228,16 +228,16 @@ export class TableState {
       // (We'll add to this after figuring out which columns are new vs existing.)
       const columnIdsToExpand = isInitial ? localStorageColumns : [];
```

Comment:
**question:** I was thinking that `columnIdsToExpand` would initialize with `this.expandedColumnIds` if `!isInitial`. Would that work here?

Reply:
Yes, it does work